### DISCUSSION TITLE
Is the item bug fixed (not spawn items)

### DISCUSSION THREAD

**Discussion by Dylised** [JaylyDev/manhunt#9] [General]
I need to know (rand item)

**Comment by JaylyDev** [comment]
I think so, haven't tested it out. Here's changelog I posted on discord:
- Minecraft crashes due to directory path link exceed
- Some low-end devices aren’t able to play some gamemodes
- Fix critical bug in randomitem mode

**Reply by Dylised** [reply]
Not fixed

**Reply by bot174** [reply]
what’s that bug and evidence lol

**Reply by Dylised** [reply]
I cant get evidence at this exact moment but the bug is for the category in subpacks random_item challenge gamemode
There is a bug so after 2 minutes when you are supposed to get stacks of random items they don't appear, neither does the text "Item Drops have arrived" (something like that)
I don't really know what the cause of this bug is, I will investigate.
But if any other 'Developers' know how to fix this can you please help,
The cause could be from the temp.txt file. I doubt it though.
Please help me investigate this code error! :)
